config/environments/test.rb

```diff
@@ -41,12 +41,12 @@
   config.serve_static_assets = true
   config.static_cache_control = "public, max-age=3600"
 
-  # Raise exception on mass assignment protection for Active Record models
-  config.active_record.mass_assignment_sanitizer = :strict
-
   #enables a few aliases - context, should, and should_eventually methods
   config.minitest_spec_rails.mini_shoulda = true
 
   # Use separate cache stores for parallel_tests
   config.cache_store = :file_store, Rails.root.join("tmp", "cache", "paralleltests#{ENV['TEST_ENV_NUMBER']}")
+  config.active_record.mass_assignment_sanitizer = :strict
+
+  config.active_record.maintain_test_schema = false
```

Comment:
Okay, can you put something short to that effect here please?  Probably just to say that this disables automatic creation of the test DB so that it has to be explicitly generated before running tests.

`rake test` still works OK though?